sphinx_rtd_theme/layout.html

```diff
@@ -28,17 +28,40 @@
     <link rel="canonical" href="{{ theme_canonical_url }}{{ pagename }}.html"/>
   {% endif %}
 
-  {# CSS #}
-
-  {# OPENSEARCH #}
-  {% if not embedded %}
-    {% if use_opensearch %}
-      <link rel="search" type="application/opensearchdescription+xml"
-            title="{% trans docstitle=docstitle|e %}Search within {{ docstitle }}{% endtrans %}"
-            href="{{ pathto('_static/opensearch.xml', 1) }}"/>
+  {# Javascript #}
```

Comment:
Nitpick: Use `{# JAVASCRIPT #}` or `{# SCRIPTS #}` in all uppercase for consistency with other section comments